Categorifier/C/Client/Internal.hs

```diff
@@ -30,6 +32,28 @@ import qualified Language.Haskell.TH.Syntax as TH
 -- so we can use @MagicHash@ for the `W8#` constructor.
 {-# ANN module "HLint: ignore Avoid restricted extensions" #-}
 
+#if __GLASGOW_HASKELL__ >= 900
+type TyVarBndr = TH.TyVarBndr ()
```

Comment:
There is a synonym for this in categorifier-th. And the patterns for `PlainTV` and `KindedTV` could live there, too ... right now there's a conditionalized `tyVarBndrName` up there, but if the TV patterns move up, then that doesn't have to be conditional.

Reply:
If this is moved to `categorifier-th`, then ideally `PlaintV` and `KindedTV` patterns should take the `flag` parameter. But I couldn't make it work (attempt: https://github.com/con-kitty/categorifier/pull/40) Do you have any suggestions?